apps/azure-functions-api/src/config.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Config module
+ *
+ * Single point of access for the application confguration. Handles validation on required environment variables.
+ * The configuration is evaluate eagerly at the first access to the module. The module exposes convenient methods to access such value.
+ */
+
+import * as t from "io-ts";
+
+import * as E from "fp-ts/lib/Either";
+import { pipe } from "fp-ts/lib/function";
+
+import * as reporters from "@pagopa/ts-commons/lib/reporters";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { withDefault } from "@pagopa/ts-commons/lib/types";
+import { NumberFromString } from "@pagopa/ts-commons/lib/numbers";
+
+export type IConfig = t.TypeOf<typeof IConfig>;
```

Comment:
Can we drop the "I" prefix from types and variables?

Reply:
Fixed in e101915